client/packages/system/src/Stock/Components/Ledger/useLedgerColumns.ts

```diff
@@ -108,3 +110,11 @@ const getLocalisationKey = (type: InvoiceNodeType): LocaleKey => {
       return 'label.repack';
   }
 };
+
+const getNameValue = (t: TypedTFunction<LocaleKey>, name: String) => {
+  if (name == 'repack') {
+    return t('label.repack');
+  } else if (name == 'Inventory adjustments') {
+    return t('inventory-adjustment');
+  } else return name;
+};
```

Comment:
Just a small read-ability thing -- since you're returning early when a condition is met, you don't need the `else` for the later conditions. Also, since they only have one statement, you don't need the curly braces. So this could just be:
```ts
const getNameValue = (t: TypedTFunction<LocaleKey>, name: String) => {
  if (name == 'repack') return t('label.repack');
  if (name == 'Inventory adjustments') return t('inventory-adjustment');
  return name;
};
```
Which does exactly the same thing, but slightly nicer to read IMHO :)

Reply:
Thanks Carl! Will fix up as sidequest in another PR :)

Reply:
https://github.com/msupply-foundation/open-msupply/pull/5769 fix here